src/styles/button.css

```diff
@@ -86,3 +86,11 @@
 .adguard-logo:before {
     content: "\00a0" !important;
 }
+
+.adguard-assistant-button-respect-vk {
+    transform: translate3d(-70px, 0, 0);
```

Comment:
@ameshkov Partial support in IE10 refers to not supporting the transform-style: preserve-3d property. This prevents nesting 3D transformed elements. Just translate3d is worked fine in IE10 and i was check it in the Browserstack